addons/extract_i18n_strings.pl

```diff
@@ -320,6 +321,7 @@ sub extract_modules {
     const('pf::config', 'Inline triggers', [$pf::config::MAC, $pf::config::PORT, $pf::config::SSID, $pf::config::ALWAYS]);
     const('pf::config', 'Network types', [$pf::config::NET_TYPE_VLAN_REG, $pf::config::NET_TYPE_VLAN_ISOL, $pf::config::NET_TYPE_INLINE, 'management', 'other']);
     const('pf::radius_audit_log', 'RADIUS Audit Log', \@pf::radius_audit_log::FIELDS);
+    const('pf::node_option82', 'Node Options 82', [values %pf::node_option82::HEADINGS]);
```

Comment:
A better name would just be 'DHCP Option82', pf::dhcp_option82 , ...
However I would want @fdurand feedback on how problem this would cause him.